docs/manuals/core/entities/scheduling-and-job-execution.md

```diff
@@ -99,7 +99,7 @@ Here is the list of the job's attributes:
 -   `BLOCKED`: The job is blocked by inputs not ready.
 -   `PENDING`: The job is waiting for execution.
 -   `RUNNING`: The job is being executed.
--   `CANCELLED`: The job was cancelled by the user.
+-   `CANCELED`: The job was canceled by the user.
```

Comment:
Only the user can cancel a Job?
Shouldn't this be "The job was canceled."?

Reply:
Yes only the user can cancel the job. The subsequent job of the canceled job that will not run is called ABANDONED